challenges/branches_arent_just_for_birds.html

```diff
@@ -192,6 +192,9 @@
 
   <p>To fix a remote that is pointing to the wrong place you can re-set its url: <code>git remote set-url origin <URLTOYOURFORK></code>.
 
+  <h4>Authentication failed...error: 401</h4>
+  <p>Your identity could not be verified. You may have 2FA turned on in which case you must use a personal access token or push through the GitHub for Mac or GitHub for Windows which handles 2FA for you.</p>  
```

Comment:
That LGTM. I'm busy with other things right now if you want to move it to the right place and close out this PR. Thanks!